snapcraft/_help.py

```diff
@@ -44,6 +44,8 @@ def _topic_help(module_name, devel):
 
 
 def _module_help(module_name, devel):
+    if module_name == "tar-content":
```

Comment:
instead of this, can you replace all `-` to `_` like we do in the plugin loader?